Community Commitments Blog Post (#202)
SeaGL/seagl.github.io: _posts/2020-09-15-community-commitments.md

---
layout: post
title: 'SeaGL Community Commitments'
status: publish
type: post
published: true
categories: news
---

> As we enter the final months of our planning season in these tumultuous times, we wanted to take a moment to look inward. In doing so, we have refined both our [Code of Conduct] and [Mission Statement]. Below you will find some of the reasoning behind these changes. Rest assured, the conference organizers have had no falling outs or great shifts of opinion. Our intent here is to be as open and candid as possible.

Reviewer: ```suggestion
As we enter the final months of our planning season in these tumultuous times, we wanted to take a moment to look inward. In doing so, we have refined both our [Code of Conduct] and [Mission Statement]. Below you will find some of the reasoning behind these changes. Rest assured, the conference organizers have had no falling outs or great shifts of opinion. Our intent here is to be as open and candid as possible.
```
```suggestion
As we enter the final months of our planning season in these tumultuous times, we wanted to take a moment to look inward. In doing so, we have refined both our [Code of Conduct](https://seagl.org/code_of_conduct.html) and Mission Statement. Below you will find some of the reasoning behind these changes. Rest assured, the conference organizers have had no falling outs or great shifts of opinion. Our intent here is to be as open and candid as possible.
```

> As we enter the final months of our planning season in these tumultuous times, we wanted to take a moment to look inward. In doing so, we have refined both our [Code of Conduct] and [Mission Statement]. Below you will find some of the reasoning behind these changes. Rest assured, the conference organizers have had no falling outs or great shifts of opinion. Our intent here is to be as open and candid as possible.

Reviewer: ```suggestion
As we enter the final months of our planning season in these tumultuous times, we wanted to take a moment to look inward. In doing so, we have refined both our [Code of Conduct] and [Mission Statement]. Below you will find some of the reasoning behind these changes. Our intent is to be as open and candid as possible.
```

> SeaGL remains committed to accessibility, inclusivity, and diversity. Our programming committee follows and makes public a [Code of Practice]. We are committed to supporting the [Black Lives Matter] movement and other social causes that align with our ethos.

Reviewer: ```suggestion
SeaGL remains committed to accessibility, inclusivity, and diversity. Our programming committee follows and makes public a [Code of Practice](https://seagl.org/news/2020/07/13/code_of_practice.html). We are committed to supporting the [Black Lives Matter](https://blacklivesmatter.com/) movement and other social causes that align with our ethos.
```

Beyond words, we are continually lowering barriers to participation with offerings such as:

> * holding office hours to help prospective speakers with their proposals as part of our CFP process

Reviewer: ```suggestion
* Holding office hours to help prospective speakers with their proposals as part of our CFP process
```

> * [offering no cost childcare and lunch at SeaGL](https://seagl.org/news/2018/10/04/Free_Childcare_Free_Lunch.html)

Reviewer: ```suggestion
* [Offering no cost childcare and lunch at SeaGL](https://seagl.org/news/2018/10/04/Free_Childcare_Free_Lunch.html)
```

> * [reaching out directly to underserved communities](???)

Reviewer: ```suggestion
* Reaching out directly to underserved communities to encourage speaking and attending
```

Author: Hmm, how do I add/change the suggestion to s/attending/attendance/ ?

Reviewer: Done :)

> * actively encouraging new speakers 

Reviewer: This one feels a little hand-wavey, and is indicated by the points above. I think we can cut it.

Author: Which one?

Author: Oh, I see it now. I think this is one of the more important ones actually, it just needs to be expanded upon. This is where we talk about selecting a novice keynotist, having office hours, etc.

> Most recently, we are planning [real-time transcription of presentations](???) for SeaGL 2020.

Reviewer: ```suggestion
Most recently, we are planning real-time transcription of presentations for SeaGL 2020.
```

> We find reward in seeing people participate who might not have done so.

Reviewer: ```suggestion
We find reward in ensuring everyone can attend SeaGL and have a positive experience.
```

Reviewer: Or something? I'm not sure who "people who might not have attended but did" is meant to be, but I see what you're going for.

However, things do not always go as smoothly as intended, and when issues arise we are committed to trying our best. We enforce a strict [Code of Conduct], making decisions collectively after gathering the necessary details. To respect the privacy of those involved and insulate any legal concerns, SeaGL's policy is to not make public statements about individual cases. While not perfect, we take specific allegations very seriously and follow up as soon as they are brought to our attention.

> All that said, the Seattle GNU/Linux conference is a big and little F free grassroots technical event organized by volunteers. It is difficult to express how much we appreciate all of those who have participated in bringing SeaGL into its 8th year (THANK YOU!). We wouldn't be here without the community that has supported us along the way and we are committed to supporting it!

Reviewer: ```suggestion
All that said, the Seattle GNU/Linux conference is a big-and-little-F free grassroots technical event organized by volunteers. It is difficult to express how much we appreciate all of those who have participated in bringing SeaGL into its 8th year (THANK YOU!). We wouldn't be here without the community that has supported us along the way and we are committed to supporting it!
```
